src/components/Highlighter.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <span
+    :class="suit()"
+  >
+    <component
+      v-for="({ value, isHighlighted }, index) in parsedHighlightedValue"
+      :class="[isHighlighted && suit('highlighted')]"
+      :key="index"
+      :is="isHighlighted ? highlightedTagName : textNode"
```

Comment:
Why not span as non highlighted?

Reply:
François is right. In React InstantSearch, span is used for non highlighted, but not in the others. Just text node.